SPECS/kernel/kernel.spec

```diff
@@ -120,16 +129,15 @@ make mrproper
 
 %ifarch x86_64
 cp %{SOURCE1} .config
-arch="x86_64"
-archdir="x86"
 %endif
 
 %ifarch aarch64
 cp %{SOURCE2} .config
-arch="arm64"
-archdir="arm64"
 %endif
 
+arch=%{arch}
+archdir=%{archdir}
+
 cp .config current_config
 sed -i 's/CONFIG_LOCALVERSION=""/CONFIG_LOCALVERSION="-%{release}"/' .config
 make LC_ALL=  ARCH=${arch} oldconfig
```

Comment:
>${arch} [](start = 19, length = 7)

I've just noticed these. I think we turn them to `%{arch}`, right? There's this one and one more below.